corgi/tasks/prod_defs.py

```diff
@@ -453,35 +435,28 @@ def parse_errata_info(
                     .first()
                 )
 
-                product_variant, _ = ProductVariant.objects.update_or_create(
+                product_variant, created = ProductVariant.objects.update_or_create(
                     name=variant,
                     defaults={
-                        "version": "",
-                        "description": "",
                         "cpe": et_variant_cpe if et_variant_cpe else "",
-                        "products": product,
-                        "productversions": product_version,
-                        "productstreams": product_stream,
                         "meta_attr": {
                             "et_product": et_product_name,
                             "et_product_version": et_pv_name,
                         },
                     },
                 )
-                node, node_created = ProductNode.objects.get_or_create(
+                node, node_created = ProductNode.objects.update_or_create(
                     parent=product_stream_node,
                     object_id=product_variant.pk,
```

Comment:
Can't use obj here because it causes a failure related to reverse lookup of relationship. That's why I've split obj to object_id, and content_type. Although the content_type here should never change.

Reply:
Ok, I made this change, and also added a test for the scenario where an inferred variant is changed to an direct one.